homewor6/task.js

```diff
@@ -0,0 +1,17 @@
+function quadraticEquation(a, b, c) {
+  const x1 = (-1 * b + Math.sqrt(Math.pow(b, 2) - 4 * a * c)) / (2 * a);
```

Comment:
do that calculation after all condition. Otherwise it will always calculate even if we have some missing parameters